cmd/operator-sdk/alpha/olminit/cmd.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olminit
+
+import (
+	"context"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/internal/olm"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+var (
+	version string
```

Comment:
This would be a good candidate for moving CLI flags into a `OLMInitCmd` struct exported from `internal/olm` that has a `Run() error` method containing code from `initOLM()`. This shouldn't necessarily be exported to users because OLM isn't unique to the SDK.

Reply:
@estroz I ended up adding `init`, `reset`, and `status` that uses a single `Manager` struct containing `Version` and `Timeout` fields, since there's a bunch of shared code between the commands.

I *think* the new refactoring meets the intent of your comment, but let me know if you had something else in mind.

Reply:
Yes! Looks great on a first pass. I'll have a more thorough look when I get a chance.